src/content/en/2021/caching.md

```diff
@@ -17,4 +19,140 @@ featured_stat_3: TODO
 featured_stat_label_3: TODO
 ---
 
-## TODO
+## Introduction
+
+Over the past decade ...
+
+
+## CDN Cache adoption
+
+Talk about CDN adoptions, 14% of desktop pages use a Cloudflare CDN.
+
+{{ figure_markup(
+  image="to-do.png",
+  caption="Top CDNs adoption",
+  description="TODO",
+  chart_url="TODO",
+  sheets_gid="58739923",
```

Comment:
@rviscomi sheet `58739923` would be possible have a figure showing the top CDNs adoption.